src/Components/Products/index.js

```diff
@@ -1,10 +1,108 @@
 import React from 'react'
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
 import styles from './products.module.css';
+import Modal from '../Shared/Modal/index';
+
+const Products = (props) => {
+  const [products, setProducts] = useState([]);
+  const [showModal, setShowModal] = useState(false);
+  const [productId, setProductId] = useState(undefined);
+
+  useEffect(() => {
+    async function fetchProducts() {
+      try {
+        const response = await fetch(`${process.env.REACT_APP_API_URL}/products/`);
+        const json = await response.json();
+        setProducts(json.data);
+      } catch (error) {
+        alert('Could not GET Products.', error);
+      }}
+    fetchProducts();
+  }, []);
+
+  const deleteProduct = async (id) => {
```

Comment:
Same concept here, move this to a thunk to delete products

Reply:
To/do on other ticket